README.md

```diff
@@ -15,3 +15,36 @@ As a user, I would like to be able to render content via a default output type,
 
 ## What's next?
 An output type provides a common backbone for all of our pages, so now we'll explore how we can start adding some semantic HTML and customization. The first concept to tackle in this regard is layouts. Switch to `fusion-03` to view a completed layout.
+
+If you are continuing this training from the last user story, then keep working on your current branch USERNAME-fusion-training.
+
+If you are starting this training from the third user story (task Fusion-03), then follow the steps under "What's next?" on the README file in the (https://github.com/wapopartners/Fusion-Training-User-Stories/blob/fusion-01/README.md)[fusion-01] branch - but checkout from fusion-02 instead.
+
+## New Task: User Story 03
+As a user, I would like to add a layout to my page, in order to give my page some structure and semantic HTML.
+
+## Guide
+1. All layouts are added to the `components/layouts` folder. Change directory into this folder and create a new find called `basic.jsx`
+
+2. In `basic.jsx`, create a new React component called `BasicLayout` that retrieves the following content from `props.children`:
+```
+const [header, main, aside, footer] = props.children;
+```
+
+3. Using these content variables, create the header, main, right rail (using the `aside` element) and footer for the page and display the header, main, aside and footer content in them respectively
+
+4. Define the different section of this layout outside the definition of your component, like so:
+```
+BasicLayout.sections = ['header', 'main', 'aside', 'footer']
+```
+
+5. Finally, define the propTypes of your component:
+```
+BasicLayout.propTypes = {
+  children: PropTypes.node,
```

Comment:
We may want to inform them that they'll need to add the PropTypes library for this